OutaTime/DatePickerViewController.swift

```diff
@@ -0,0 +1,32 @@
+//
+//  DatePickerViewController.swift
+//  OutaTime
+//
+//  Created by Nichole Davidson on 1/29/20.
+//  Copyright © 2020 Nichole Davidson. All rights reserved.
+//
+
+import UIKit
+
+protocol DatePickerDelegate {
+    
+    func destinationDateWasChosen(_: Date)
+}
+
+class DatePickerViewController {
+    
+    @IBOutlet weak var datePicker: UIDatePicker!
+    
+    @IBAction func cancelTapped(_ sender: UIBarButtonItem) {
+        dismiss(animated: true, completion: nil)
+    }
+    @IBAction func doneTapped(_ sender: UIBarButtonItem) {
+        
+        delegate?.doneTapped(_: DateFormatter)
```

Comment:
Needed to call the `destinationDateWasChosen` here.
EX: `delegate?.destinationDateWasChosen(datePicker.date)`